app/controllers/application_controller.rb

```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+  before_action :authenticate_user!
```

Comment:
@devc0der20 - It looks like the gardens#index is blocked by devise.  I saw you put skip_before_action in the gardens_controller but for me it's still blocking the home page if you're not a logged in user.  Maybe the before_action :authenticate_user! method can just go in the top of the bookings controller because that's the only place it's needed?  One line of code vs two also ;)

Reply:
well tested locally and index was and show were working without any authentication needed. tested several times.

Reply:
Well, I pulled your branch and tested it locally on my machine and it didn't work. Are you up to date with master and merged master into your branch? 
`git merge master`
Please check and fix if necessary, so that I can merge your changes into master :)